configs/vite_labs.json

```diff
@@ -1,32 +1,34 @@
 {
   "index_name": "vite_labs",
   "start_urls": [
+    {
+      "url": "http://doc.vite.org/",
+      "variables": {
+        "lang": [
+          "en"
```

Comment:
These webpages will not have the right variables since it isn't injected somewhere

Reply:
You can use tags instead.

Reply:
ok, I see.

Reply:
The need to add a new language to the site is something required.
Otherwise we can inject it from another webpage but I would rather have it written properly.

Reply:
Ok, I see